x-pack/solutions/observability/plugins/observability/public/services/telemetry/telemetry_events.ts

```diff
@@ -33,4 +33,35 @@ const alertDetailsPageView: TelemetryEvent = {
   },
 };
 
-export const events: TelemetryEvent[] = [relatedAlertsLoaded, alertDetailsPageView];
+const linkedDashboardView: TelemetryEvent = {
+  eventType: TelemetryEventTypes.LINKED_DASHBOARD_VIEW,
+  schema: {
```

Comment:
can we add these properties in schema - action type, page type, rule type. As mentioned in another comment, dashboard Id is not needed.